tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "wit-widget",
+  "version": "0.1.4",
+  "description": "What-If Tool jupyter widget",
+  "author": "Google LLC",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tensorflow/tensorboard.git"
+  },
+  "keywords": [
+    "jupyter",
+    "widgets",
+    "ipython",
+    "ipywidgets",
+    "jupyterlab-extension"
+  ],
+  "files": [
+    "lib/**/*.js",
+    "dist/*.js"
+  ],
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Two options: (1) if you want an ability to use the wit-widget JS payload (without any python interaction which I don't think it possible for this widget), you may want to include an instruction to npm publish this package, or (2) set `private: true` ([npm doc](https://docs.npmjs.com/files/package.json#license); `Consider also setting "private": true to prevent accidental publication.`)

Reply:
set private to true, thanks